### ISSUE TITLE
Rotation Invariance Trained Model

### ISSUE THREAD

**Issue by yuvalH9** [XuyangBai/D3Feat.pytorch#9]
Hi Xuyang Bai,

In the arxiv version of your paper you reported in the supplementary material section that when adding rotation augmentations in the training step you achieve rotation invariance.

Did the supplied trained model on this repository was created using augmentations, i.e. should it be rotation invariant?
If no, can you supply a link to that trained model ?

What range of rotation angles did you use?

Regards,
Yuval

**Comment by XuyangBai** [comment]
Hi, yes I also use the rotation augmentation for the pytorch version so it is expected to have some robustness towards rotation, but it is not theoretically rotation invariant. The range of rotation angle for the augmentation is [0-2pi] along one random axis as shown below.

https://github.com/XuyangBai/D3Feat.pytorch/blob/cb44a16657cf2da3e5167c539555c9bd90df2da8/config.py#L81-L84
